spec/models/curl_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Curl, type: :model do
+  describe 'CA Certificate Works' do
+    it 'does not throw an error when curling over https' do
+      expect do
+        Curl::Easy.perform('https://services.etosoftware.com/') do |http|
+          # http.cacert = '/etc/ssl/certs/ca-certificates.crt'
+          http.verbose = true
+        end
+      end.to_not raise_error
```

Comment:
I wonder if you should 'x' this test, and copy it and flip the sense around on this so this passes as long as the work around is required?

Reply:
Oh, this test is supposed to fail for right now.  I'm testing some other mechanisms (I'd really like to set the CA Path in an initializer if possible.)